.github/workflows/pypi-cd.yml

```diff
@@ -68,25 +69,30 @@ jobs:
                   cat Cargo.toml
 
             - name: Set up Python
-              if: matrix.build.PYPI_PUBLISH == true && !contains(matrix.build.RUNNER, 'self-hosted')
+              if: ${{ !contains(matrix.build.RUNNER, 'self-hosted') }}
```

Comment:
minor: why do you need the curly braces here and not in lines 78 / 83?

Reply:
The linter gets angry when it's with ! (not) :(